components/brave_today/browser/suggestions_controller.cc

```diff
@@ -253,21 +255,28 @@ SuggestionsController::GetSuggestedPublisherIdsWithHistory(
     }
   }
 
-  std::vector<std::string> result;
+  std::vector<std::string> all_suggestions;
   for (const auto& [publisher_id, score] : scores) {
     // Either the source it was similar to was disabled, or
     // the source is already enabled.
     if (score == 0)
       continue;
-    result.push_back(publisher_id);
+    all_suggestions.push_back(publisher_id);
   }
 
-  std::sort(result.begin(), result.end(),
+  std::sort(all_suggestions.begin(), all_suggestions.end(),
             [scores](const std::string& a_id, const std::string& b_id) {
               return scores.at(a_id) > scores.at(b_id);
             });
 
-  return result;
+  auto suggestions_count = std::min(all_suggestions.size(), kMaxSuggestions);
```

Comment:
@sangwoo108 is there a better way to do this? I basically want to truncate the sources to `kMaxSuggestions`

Reply:
Cool, I think your code is describing itself well, so good enough. If you would want to simpler one, I think you might want to use std::vector.resize().

```
if (all_suggestions.size() > kMaxSuggestions)
  all_suggestions.resize(kMaxSuggestions);
```

Reply:
Perfect, this is exactly what I wanted. Thanks @sangwoo108!